pkgs/development/compilers/llvm/common/llvm/default.nix

```diff
@@ -307,8 +307,8 @@ stdenv.mkDerivation (finalAttrs: {
       ; do
       echo "PATCH: $f"
       substituteInPlace $f \
-        --replace-fail 'Starting llvm::' 'Starting {{.*}}' \
```

Comment:
Since this is only applicable to version 12 (see line 292) shouldn't we just remove the lines where version 12 doesn't have the string?

I need to do a build to see what they are, but 12 is the oldest version of clang/llvm in nixpkgs now.

Reply:
Oh, I had missed that this was only for 12. I'm easy either way with keeping this as (after "-quiet") is or discarding it. Given that it is legacy and no more harmful than it was before the switchover to replace-fail, I'm inclined to not expend too much more effort in this area other than to make it work. I'd appreciate if you're doing a build anyway to see what is going on if you can fix this PR up or make your own as you find easier.

Reply:
Yeah, it's certainly not worth expending a lot of effort here, for sure. I'm in process of ascertaining and will update.